---
float32 east	# east coordinate in Lake Lag frame in meters of the truckbed
float32 north	# north coordinate in Lake Lag frame in meters of the truckbed